readium/navigator/src/main/java/org/readium/r2/navigator/media/ExoMediaPlayer.kt

```diff
@@ -114,7 +114,7 @@ class ExoMediaPlayer(
         .apply {
             setMediaSessionToken(mediaSession.sessionToken)
             setPlayer(player)
-            setSmallIcon(R.drawable.exo_notification_small_icon)
+            setSmallIcon(com.google.android.exoplayer2.ui.R.drawable.exo_notification_small_icon)
```

Comment:
We should not use unqualified `R` when referencing resources of nested dependencies, as it prevent using `android.nonTransitiveRClass=true`.

`android.nonTransitiveRClass=true` is enabled by default in Android Studio's project template and improves build performance.